presto-main/src/main/java/io/prestosql/operator/WindowOperator.java

```diff
@@ -628,25 +635,45 @@ void updateMemoryUsage()
 
             // If we have unused input or are finishing, then we have buffered a full group
             if (finishing || pendingInputPosition < pendingInputOptional.get().getPositionCount()) {
-                sortPagesIndexIfNecessary(inMemoryPagesIndexWithHashStrategies, orderChannels, ordering);
-                // TODO spill if updating moving from revocable memory to regular is not possible
-                updateMemoryUsage(false);
```

Comment:
remove `revocablePagesIndex` parameter from `updateMemoryUsage`

Reply:
How would we know if pages index is revocable then?